source/plot/PlotData2D.ooc

```diff
@@ -3,67 +3,17 @@ use ooc-collections
 use ooc-math
 
 PlotData2D: abstract class {
-	label: String { get set }
-	lineWidth: Float { get set }
-	legendOffset: Float { get set }
-	colorBgra: ColorBgra { get set }
-	color: String { get set }
-	opacity: Float { get set }
-	dataSeries: VectorList<FloatPoint2D> { get set }
-
-	init: func ~default {
-		this init(VectorList<FloatPoint2D> new())
-	}
-
-	init: func ~dataSeries (=dataSeries, label := "", colorBgra := ColorBgra new()) {
-		this lineWidth = 1
-		this legendOffset = 5.0f
-		this label = label
-		this colorBgra = colorBgra
-	}
-
-	init: func ~color (dataSeries: VectorList<FloatPoint2D>, colorBgra: ColorBgra) {
-		this init(dataSeries, "", colorBgra)
-	}
-
-	init: func ~twoFloatSeries (xSeries, ySeries: VectorList<Float>, label := "", colorBgra := ColorBgra new()) {
-		dataSeries := VectorList<FloatPoint2D> new()
-		for (i in 0 .. ySeries count) {
-			dataSeries add(FloatPoint2D new(xSeries != null ? xSeries[i] : (i + 1) as Float, ySeries[i]))
-		}
-		this init(dataSeries, label, colorBgra)
-	}
-
+	label: String
+	lineWidth: Float = 1.0f
+	legendOffset: Float = 5.0f
+	colorBgra: ColorBgra
```

Comment:
Isn't this PR going to conflict with your other one ? I'm pretty sure I saw this change yesterday in your other PR